anaconda.spec.in

```diff
@@ -196,8 +196,6 @@ Requires: hfsplus-tools
 %ifnarch riscv64
 Requires: kexec-tools
 %endif
-# needed for proper driver disk support - if RPMs must be installed, a repo is needed
-Requires: createrepo_c
```

Comment:
Maybe `generate_driver_disk_repositories` should check something like `conf.system.supports_driver_disks` that will be enabled only for a boot.iso. The `CopyDriverDisksFilesTask` class should do the same. It would be better to not run this code at all if we think it shouldn't be necessary.

Reply:
Thanks, added.